utilities/titandb/blob_file_iterator.h

```diff
@@ -0,0 +1,75 @@
+//
+// Created by 郑志铨 on 2018/8/9.
+//
+
+#ifndef ROCKSDB_BLOB_FILE_ITERATOR_H
+#define ROCKSDB_BLOB_FILE_ITERATOR_H
+
+#include <stdint.h>
+#include "include/rocksdb/slice.h"
+#include "include/rocksdb/status.h"
+#include "table/internal_iterator.h"
+#include "util/file_reader_writer.h"
+#include "utilities/titandb/blob_format.h"
+
+namespace rocksdb {
+namespace titandb {
+
+class BlobFileIterator final : public InternalIterator {
```

Comment:
I think it is not a good idea to inherit the iterator interface because the interface is far different from what you need.

Reply:
Yes, you are right. I will change it ASAP